addons/account_edi_ubl_cii/data/ubl_20_templates.xml

```diff
@@ -198,22 +198,27 @@
             <cbc:ID t-out="vals.get('id')"/>
             <cbc:Note t-out="vals.get('note')"/>
             <cbc:InvoicedQuantity
-                    t-if="invoice.move_type == 'out_invoice'"
+                    t-if="document_type == 'invoice'"
                     t-att="vals.get('invoiced_quantity_attrs', {})"
                     t-out="vals.get('invoiced_quantity')"/>
             <cbc:CreditedQuantity
-                    t-if="invoice.move_type == 'out_refund'"
+                    t-if="document_type == 'credit_note'"
+                    t-att="vals.get('invoiced_quantity_attrs', {})"
+                    t-out="vals.get('invoiced_quantity')"/>
+            <cbc:DebitedQuantity
```

Comment:
I would create a template `ubl_20_DebitNoteLineType` inheriting `ubl_20_InvoiceLineType` with primary=True to add the `DebitedQuantity` node. This way we avoid having DebitedQuantity inside the InvoiceLine template.
The difference between the `DebitNoteLine` and the `InvoiceLine` nodes in UBL 2.0 is simply: `{'DebitedQuantity', 'DiscrepancyResponse'}`.

Reply:
Same response as previous comment :)